src/models/chan.js

```diff
@@ -259,6 +259,6 @@ Chan.prototype.isLoggable = function() {
 	return this.type === Chan.Type.CHANNEL || this.type === Chan.Type.QUERY;
 };
 
-Chan.prototype.requestZncPlayback = function(network, from) {
-	network.irc.raw("ZNC", "*playback", "PLAY", this.name, from.toString());
-};
+function requestZncPlayback(channel, network, from) {
```

Comment:
Could do `requestZncPlayback({name}, network, from)` or passing `name` directly `requestZncPlayback(name, network, from)` but that is awfully nitpicky :D

Reply:
Just `name` is more confusing because it doesn't tell you what name it should be, passing `channel` is more clearer, imo.

Reply:
Fair enough.